### PR TITLE
Feature/29 Adjust transaction detail page style

### PR DESCRIPTION
**Pull request by ylembachar**
Adds the header, sidebar and transaction detail first styles.

### CODE REVIEW

File: frontend/cypress/components/Transaction.cy.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {transactionData, updatedTransactionData, verifyTransactionDetails} from
 
 describe('<Transaction />', () => {
     it('renders the transaction details correctly', () => {
-        cy.intercept('GET', '/api/transaction/*', {
+        cy.intercept('GET', `/api/transaction/hash=${transactionData.userTransactionHash}`, {
```
Comment (fredo) [inline frontend/cypress/components/Transaction.cy.tsx:9]: Is this intended?

File: frontend/cypress/components/Transaction.cy.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ describe('<Transaction />', () => {
     });
 
     it('updates the transaction details after polling', () => {
-        cy.intercept('GET', `/api/transaction/${transactionData.userTransactionHash}`, {
+        cy.intercept('GET', `/api/transaction/hash=${transactionData.userTransactionHash}`, {
```
Comment (fredo) [inline frontend/cypress/components/Transaction.cy.tsx:26]: Didn't we change it to `api/transaction/HASH` just in the previous PR?

Comment (ylembachar) [reply frontend/cypress/components/Transaction.cy.tsx]: I had just removed the additional transaction keyword in the URL in the previous PR, but the correct API call is anyway without hash= so I've updated it now.

**Review by fredo — APPROVED** [review]
Please add the correct package.json and package-lock.json before merging. Otherwise looks good!

**Pull request merged**